platforms/documentation/docs/src/snippets/bestPractices/useConventionPlugins-do/groovy/build-logic/src/main/groovy/my.java-common-conventions.gradle

```diff
@@ -0,0 +1,31 @@
+// tag::do-this[]
+plugins {
+    id("java-library")
+}
+
+repositories {
+    mavenCentral()
+}
```

Comment:
Shouldn't this part be moved into `settings.gradle` as described in https://docs.gradle.org/current/userguide/bp_dependencies.html#set_up_repositories_in_settings?

Reply:
Good idea, we should update this.